connexion/apps/quart_app.py

```diff
@@ -0,0 +1,119 @@
+        app.json_encoder = QuartJSONEncoder
+        return app
+
+    def get_root_path(self):
+        return pathlib.Path(self.app.root_path)
+
+    def set_errors_handlers(self):
+        # Quart uses hypercorn, which doesn't have exceptions like werkzeug
+        for error_code in quart.exceptions.default_exceptions:
+            self.add_error_handler(error_code, self.common_error_handler)
+
+        self.add_error_handler(ProblemException, self.common_error_handler)
+
+    @staticmethod
+    async def common_error_handler(exception):
+        """
+        :type exception: Exception
+        """
+        if isinstance(exception, ProblemException):
+            response = problem(
+                status=exception.status, title=exception.title, detail=exception.detail,
+                type=exception.type, instance=exception.instance, headers=exception.headers,
+                ext=exception.ext)
+        else:
+            if not isinstance(exception, quart.exceptions.HTTPException):
+                exception = quart.exceptions.HTTPStatusException()
+
+            response = problem(title=exception.name, detail=exception.description,
+                               status=exception.status_code)
+
+        return await QuartApi.get_response(response)
+
+    def add_api(self, specification, **kwargs):
+        api = super().add_api(specification, **kwargs)
+        self.app.register_blueprint(api.blueprint)
+        return api
+
+    def add_error_handler(self, error_code, function):
+        # type: (int, FunctionType) -> None
+        self.app.register_error_handler(error_code, function)
+
+    def run(self, port=None, server=None, debug=None, host=None, **options):  # pragma: no cover
+        """
+        Runs the application on a local development server.
+        :param port: port to listen to
+        :type port: int
+        :param server: which asgi server to use
+        :type server: str | None
+        :param debug: include debugging information
+        :type debug: bool
+        :param host: the host interface to bind on.
+        :type host: str
+        :param options: options to be forwarded to the underlying server
+        :type options: Any
+        """
+        # this function is not covered in unit tests because we would effectively testing the mocks
+
+        # overwrite constructor parameter
+        if port is not None:
+            self.port = port
+        elif self.port is None:
+            self.port = 5000
+
+        self.host = host or self.host or '0.0.0.0'
+
+        if server is not None:
+            self.server = server
+
+        if debug is not None:
+            self.debug = debug
+
+        logger.debug('Starting %s HTTP server..', self.server, extra=vars(self))
+        if self.server == 'hypercorn':
+            self.app.run(self.host, port=self.port, debug=self.debug, **options)
+        else:
+            raise Exception('Server {} not recognized'.format(self.server))
```

Comment:
Seems like you can run with different ASGI servers as well.
https://pgjones.gitlab.io/quart/tutorials/deployment.html

Probably ok to start with Hypercorn, but Uvicorn might be nice to compare with FastAPI.